assignment1_calculator.c

```diff
@@ -0,0 +1,125 @@
+// Kalpvriksha Program | Batch-2025: C Programming Assignments covering Calculator.
+
+// assignment 1: Calculator, input format :- 3+5
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+#define MAX_EXPR_LEN 100
+
+typedef enum {
+    NO_ERROR,
+    DIVISION_BY_ZERO,
+    INVALID_EXPRESSION
+} ErrorType;
+
+ErrorType error = NO_ERROR;
+
+char* skip_whitespace(char *expr) {
+    while (*expr == ' ') expr++;
+    return expr;
+}
+
+int evaluate_term(char **expr);
```

Comment:
apply code formatting and match function names with this standards, take care of this at all places.

Use camelcase for function names.
Start function names with a verb to indicate action.
Keep names concise but meaningful.
ex :
int calculateSum(int a, int b);
void displayMessage(const char *message);